pages/home/HomePage.jsx

```diff
@@ -58,18 +60,17 @@ var HomePage = React.createClass({
   handleClubBtnClick: function() {
     ga.event({ category: 'Clicked Home CTA', action: 'Start A Mozilla Club' });
   },
-  render: function() {
-    return (
-      <div>
-        <HeroUnit>
-          <h1><FormattedMessage id="MLN" /></h1>
-          <IconButtons>
-            <IconButton
-              imgSrc="/img/pages/home/svg/icon-newsletter.svg"
-              head={this.context.intl.formatMessage({id: 'get_email_update'})}
-              onClick={this.handleEmailBtnClick}
-              className={"newsletter"}
-            />
+  renderIconButtons() {
+    let newsletterCTA = !HIDE_NEWSLETTER_SIGNUP_FORM ?
+                          <IconButton
+                            imgSrc="/img/pages/home/svg/icon-newsletter.svg"
+                            head={this.context.intl.formatMessage({id: 'get_email_update'})}
+                            onClick={this.handleEmailBtnClick}
+                            className={"newsletter"}
+                          /> : null;
+
+    return <IconButtons>
+            { newsletterCTA }
```

Comment:
Just put the stuff at line 64 inline here since it's only used once. No need for a variable.

For conditional JSX like this where you only have content for an `if` case, use [this syntax](https://facebook.github.io/react/docs/conditional-rendering.html#inline-if-with-logical--operator) instead of a ternary. It's easier to read and maintain.